event_reporter/reporter/event_payload.go

```diff
@@ -128,6 +128,16 @@ func getResourceEventPayload(
 		errors = append(errors, parseApplicationSyncResultErrorsFromConditions(originalApplication.Status)...)
 	}
 
+	if originalApplication != nil && originalApplication.Status.Resources != nil {
+		for _, rs := range originalApplication.Status.Resources {
```

Comment:
please add comment why we need this

Reply:
also check the use case if app has 2 child apps and rollout for example -- if it will report the error or not